### PR TITLE
Tweak: System Info - New experiments reporter. ED-1938

### PR DESCRIPTION
**Pull request by iNewLegend**
image

**This is only log reporter, the tracking data is already exist.**
The reported created it by @udidol i just make it to work with tracking data.

### CODE REVIEW

**Review by matipojo — CHANGES_REQUESTED** [review]
Please update the JSON schema.

**Comment by iNewLegend** [comment]
@matipojo
> Please update the JSON schema.

it was updated here: https://github.com/elementor/elementor/pull/13406/

this only log reporter see https://github.com/elementor/elementor/pull/14800/files#diff-b21c1bcebdf04abc54e1aeaf2161eafb86bb97d211ebb58f1105d160b3abdb24R68-R75

**Review by matipojo — CHANGES_REQUESTED** [review]
Please pull from 3.4.0

File: core/experiments/manager.php [hunk 1]
```diff
@@ -448,6 +445,28 @@ private function get_feature_settings_label_html( array $feature ) {
 		return ob_get_clean();
 	}
 
+	/**
+	 * Get Feature State Label
+	 *
+	 * @since 3.3.0
```
Comment (matipojo) [inline core/experiments/manager.php:451]: 3.4.0

File: includes/tracker.php [hunk 2]
```diff
@@ -223,7 +223,7 @@ public static function set_opt_in( $value ) {
 	 * @return array The data from system reports.
 	 */
 	private static function get_system_reports_data() {
-		$reports = Plugin::$instance->system_info->load_reports( System_Info\Main::get_allowed_reports() );
+		$reports = Plugin::$instance->system_info->load_reports( System_Info\Main::get_allowed_tracked_reports() );
```
Comment (matipojo) [inline includes/tracker.php:226]: Replace all `tracked` with `usage`

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/experiments/manager.php (hunk 1 above)
Comment (iNewLegend) [reply core/experiments/manager.php]: Handled.

File: includes/tracker.php (hunk 2 above)
Comment (iNewLegend) [reply includes/tracker.php]: Handled.

**Review by matipojo — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please pull from develop

File: modules/system-info/reporters/experiments.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+ */
+class Experiments extends Base {
+
+	/**
+	 * Get experiments reporter title.
+	 *
+	 * @return string Reporter title.
+	 */
+	public function get_title() {
+		return 'Elementor Experiments';
```
Comment (StyleShit) [inline modules/system-info/reporters/experiments.php:20]:
```suggestion
		return esc_html__( 'Elementor Experiments', 'elementor' );
```

File: modules/system-info/reporters/experiments.php [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+	public function get_experiments() {
+		$experiments = Plugin::$instance->experiments->get_features();
+
+		$output = '';
+
+		foreach ( $experiments as $experiment ) {
+			// If the state is default, add the default state to the string.
+			$state = Plugin::$instance->experiments->get_feature_state_label( $experiment );
+
+			$output .= '<tr><td>' . $experiment['title'] . ': </td>';
```
Comment (StyleShit) [inline modules/system-info/reporters/experiments.php:55]: It should be escaped?
```suggestion
			$output .= '<tr><td>' . esc_html( $experiment['title'] ) . ': </td>';
```

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: modules/system-info/reporters/experiments.php (hunk 3 above)
Comment (iNewLegend) [reply modules/system-info/reporters/experiments.php:20]: done.

File: modules/system-info/reporters/experiments.php (hunk 4 above)
Comment (iNewLegend) [reply modules/system-info/reporters/experiments.php]: done.

File: modules/system-info/reporters/experiments.php (hunk 3 above)
Comment (StyleShit) [reply modules/system-info/reporters/experiments.php:20]: I don't see the change

**Review by StyleShit — CHANGES_REQUESTED** [review]

**Review by StyleShit — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
PHP Lint

**Pull request merged**